code/modules/mob/living/simple_animal/abnormality/aleph/seasons.dm

```diff
@@ -95,10 +98,10 @@
 		)
 
 	var/list/modular_damage_coeff = list(
-		"spring" = list(RED_DAMAGE = 0.8, WHITE_DAMAGE = 0.2, BLACK_DAMAGE = 1, PALE_DAMAGE = 1.5),
-		"summer" = list(RED_DAMAGE = 0.1, WHITE_DAMAGE = 0.6, BLACK_DAMAGE = 0.6, PALE_DAMAGE = 1), //Summer is tanky
-		"fall" = list(RED_DAMAGE = 0.8, WHITE_DAMAGE = 1, BLACK_DAMAGE = 0.2, PALE_DAMAGE = 1.5),
-		"winter" = list(RED_DAMAGE = 1.5, WHITE_DAMAGE = 1, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 0.2)
+		"spring" = list(RED_DAMAGE = 0.8, WHITE_DAMAGE = 0.2, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 1),
```

Comment:
Most ALEPH's have 1.2 weakness to a damage type at most, and for good reason. Raising it to 1.5 will make this abnormality too easy to fight.